sdk/bootstrap/main.go

```diff
@@ -29,6 +29,11 @@ const (
 
 var verbose = false
 
+// The following replaces the keys with the values when logging.
+var blackListedEnvironKeys = map[string]string{
+	"DCOS_SERVICE_ACCOUNT_CREDENTIAL": "********",
```

Comment:
We technically don't need the second part of the map. This will be the same for all entries.

Reply:
It will be same, but, I added it to understand if `DCOS_SERVICE_ACCOUNT_CREDENTIAL` is present (or not). Should we not retain that information? fwiw, the name of those environment variables will be quite obvious by looking at the the publicly available `config.json` and `marathon.json.mustache` but the presence (or absence) is not - and I am not sure if we should have this information in the logs to make it easier for debugging.

Reply:
I mean we could still check for presence in the map, it's just that we wouldn't store or use the `value`. So here we would have:
```golang
var blackListedEnvironKeys = map[string]bool{
	"DCOS_SERVICE_ACCOUNT_CREDENTIAL": true,
}
```
and then where we use it we would then have:
```golang
if _, ok := blackListedEnvironKeys[key]; ok {
    filteredEnviron = append(filteredEnviron, fmt.Sprintf("%s=**********", key))
}
```